src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionjavapatchfilter/AvoidNestedBlocks/context/defaultContextConfig.xml

```diff
@@ -9,7 +9,7 @@
         <module name="com.puppycrawl.tools.checkstyle.filters.SuppressionJavaPatchFilter">
             <property name="file" value="${tp}/defaultContext.patch" />
             <property name="strategy" value="context" />
-            <property name="neverSuppressedChecks" value="AvoidNestedBlocksCheck" />
+            <property name="checkNamesForContextStrategyByTokenOrAncestorSet" value="AvoidNestedBlocksCheck" />
```

Comment:
No difference with neverSuppressedChecks results. We should add violation which will be suppressed after this update. Please add another method with a nested block (to Test.java), which produces a violation, but will be suppressed by patch-filter.